test/stress_tests/run_application_stress_tests.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# Cause the script to exit if a single command fails.
+set -e
+
+# Show explicitly which commands are currently running.
+set -x
+
+ROOT_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)
+
+# Start a cluster for IMPALA using the autoscaler.
+CLUSTER=$ROOT_DIR/rllib_cluster.yaml
+RLLIB_DIR=$ROOT_DIR/../../python/ray/rllib/
+
+ray up -y $CLUSTER
+
+ray rsync_up $CLUSTER $RLLIB_DIR/tuned_examples/ tuned_examples/
+ray exec $CLUSTER 'rllib train -f tuned_examples/atari-impala.yaml'
+                    ' --redis-address="localhost:6379"'
+                    ' '
```

Comment:
what is this line for?

Reply:
Thanks. This is still wip; everything will be cleaned up later.